client/src/modules/stock/entry/modals/findTransfer.modal.html

```diff
@@ -0,0 +1,53 @@
+<form 
+  name="FindForm"
+  bh-submit="$ctrl.submit(FindForm)"
+  novalidate>
+
+  <div class="modal-header space-between">
+    <div class="headercrumb">
+      <i class="fa fa-search"></i> 
+      <span translate>FORM.LABELS.SEARCH</span> 
+      <span translate>STOCK.ENTRY_TRANSFER</span>
+    </div>
+
+    <div class="toolbar text-right">
+      <div class="toolbar-item">
+        <button type="button"
+          ng-click="$ctrl.toggleFilter()"
+          class="btn btn-sm btn-default"
+          ng-class="{ 'btn-info' : $ctrl.filterEnabled }">
+          <span class="fa fa-filter"></span>
+        </button>
+      </div>
+    </div>
+  </div>
+
+  <div class="modal-body">
+    <div class="row">
+      <div class="col-xs-12">
+        <div
+          id="TransferGrid"
+          ui-grid="$ctrl.gridOptions"
+          ui-grid-auto-resize
+          ui-grid-resize-columns
+          ui-grid-selection>
+          <bh-grid-loading-indicator
+            loading-state="$ctrl.loading"
+            empty-state="$ctrl.gridOptions.data.length === 0"
+            error-state="$ctrl.hasError">
+          </bh-grid-loading-indicator>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="modal-footer">
+    <button type="button" class="btn btn-default" ng-click="$ctrl.cancel()" data-method="cancel" translate>
+      FORM.BUTTONS.CLOSE
+    </button>
+
+    <bh-loading-button loading-state="FindForm.$loading">
```

Comment:
I can submit this without any choices and it loads every lot.  Is this intended behavior?